blocks/library/paragraph/style.scss

```diff
@@ -1,13 +1,31 @@
-p.has-drop-cap {
-	&:first-letter {
-		float: left;
-		font-size: 4.1em;
-		line-height: 0.7;
-		font-family: serif;
-		font-weight: bold;
-		margin: .07em .23em 0 0;
-		text-transform: uppercase;
-		font-style: normal;
+p {
+	&.is-small-text {
```

Comment:
Should this classes be outside of paragraph so other blocks can make use of our font size mechanism?

Reply:
maybe, but I'm cautious about defining styles generically for `*.is-small-text`. Also, in some other blocks reusing this component, they might want to have more fine-grained targets.

Reply:
So this is a problem because it _is_ applying outside the paragraph block. There's no scoping on the `p` selector to make this specific to the paragraph block.